frontend/src/framework/utils/timestampUtils.ts

```diff
@@ -2,13 +2,13 @@
 // According to spec, only the hour part of time is mandatory
 export function hasTime(isoDateTimeString: string): boolean {
     // Regular expression to match time portion in ISO 8601 format
-    const regex = /T\d{2}(:\d{2})?(:\d{2}(\.\d+)?)?/;
+    const regex = /^(.*T.*)\d{2}/;
```

Comment:
I would simply suggest:
```typescript
const regex = /T\d{2}/
```

We need to know if there is a `T` and that it needs to be directly followed by at least two digits.

Your regex here would also be false positive on e.g. `2023‐09‐01T−07:00` since there is no time given (although that does not seem to be a valid ISO 8601 string; time zone can only be used in combination with time it seems, or can't it?).